meta-iot2000-example/recipes-support/swupdate/swupdate/swupdate.cfg

```diff
@@ -0,0 +1,15 @@
+globals :
+{
+    verbose  = true;
+    loglevel = 10;
+    syslog   = false;
+};
+
+processes :
+(
+    {
+        name = "progress_firmware";
```

Comment:
Well, there should be a word in the readme about this mechanism automatically rebooting after a firmware update.

Reply:
Added it in the last NOTE section.